dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/impl/NettyFrameworkImpl.java

```diff
@@ -429,9 +440,22 @@ public BootstrapExtended createUDPBootstrapOutbound(Map<String, Object> options)
     }
 
     @Override
+    @FFDCIgnore({ NettyException.class })
     public FutureTask<ChannelFuture> start(ServerBootstrapExtended bootstrap, String inetHost, int inetPort,
```

Comment:
The method signature for this method is different in http-netty-transport so I just kept it the same here until we merge the rest of that work to integration.